aom/src/main/java/com/nedap/archie/adl14/ADL14DefaultMultiplicitiesSetter.java

```diff
@@ -0,0 +1,51 @@
+package com.nedap.archie.adl14;
+
+import com.nedap.archie.aom.Archetype;
+import com.nedap.archie.aom.CAttribute;
+import com.nedap.archie.aom.CObject;
+import com.nedap.archie.base.Cardinality;
+import com.nedap.archie.base.MultiplicityInterval;
+import com.nedap.archie.rminfo.MetaModel;
+import com.nedap.archie.rminfo.MetaModels;
+
+/**
+ * Sets the default occurrences, cardinality and existence with ADL 1.4 rules, if not explicitly set
+ * in a given Archetype. Useful for conversion to ADL 2, where the default values are different, and
+ * it is good to start with the correct values already present
+ */
```

Comment:
Lets put the whole reasoning in one place and make it consistent:
```suggestion
/**
 * Sets the default occurrences with ADL 1.4 rules, if not explicitly set in a given Archetype.
 * Useful for conversion to ADL 2, where the default values are different, and it is good to start
 * with the correct values already present.
 *
 * Cardinality and existence are also specified to have a default value. However, this is not used
 * in practice (source, several openEHR community members). Adding that to the conversion would
 * lead to problems. So these are left out.
 */
```